README.md

```diff
@@ -1,565 +1,21 @@
-[Paypal badge](#top).
-
-If you used spacemacs in a project, and you want to show that fact, you can use
-the spacemacs badge: [![Built with Spacemacs](https://cdn.rawgit.com/syl20bnr/spacemacs/442d025779da2f62fc86c2082703697714db6514/assets/spacemacs-badge.svg)](http://spacemacs.org)
-
-- For Markdown:
-
-   ```markdown
-   [![Built with Spacemacs](https://cdn.rawgit.com/syl20bnr/spacemacs/442d025779da2f62fc86c2082703697714db6514/assets/spacemacs-badge.svg)](http://spacemacs.org)
-   ```
-
-- For HTML:
-
-   ```html
-   <a href="http://spacemacs.org"><img alt="Built with Spacemacs" src="https://cdn.rawgit.com/syl20bnr/spacemacs/442d025779da2f62fc86c2082703697714db6514/assets/spacemacs-badge.svg" /></a>
-   ```
-
-- For Org-mode:
-
-   ```org
-   [[http://spacemacs.org][file:https://cdn.rawgit.com/syl20bnr/spacemacs/442d025779da2f62fc86c2082703697714db6514/assets/spacemacs-badge.svg]]
-   ```
-
-Thank you!
-
-[Twitter]: http://i.imgur.com/tXSoThF.png
-[CONTRIBUTING.org]: CONTRIBUTING.org
-[CONVENTIONS.org]: http://spacemacs.org/doc/CONVENTIONS
-[DOCUMENTATION.org]: http://spacemacs.org/doc/DOCUMENTATION
-[QUICK_START.org]: http://spacemacs.org/doc/QUICK_START
-[FAQ.org]: http://spacemacs.org/doc/FAQ
-[VIMUSERS.org]: http://spacemacs.org/doc/VIMUSERS
-[dotfile]: http://spacemacs.org/doc/DOCUMENTATION#dotfile-configuration
-[osx layer]: http://spacemacs.org/layers/+os/osx/README.html
-[Gitter Chat]: https://gitter.im/syl20bnr/spacemacs
-[Gitter Chat IRC server]: https://irc.gitter.im/
-[Homebrew]: http://brew.sh
-[emacs-plus]: https://github.com/d12frosted/homebrew-emacs-plus
-[powerline documentation]: http://spacemacs.org/doc/DOCUMENTATION.html#powerline-separators
-[emacs-for-windows]: https://ftp.gnu.org/gnu/emacs/windows/
-[gzip-for-windows]: http://gnuwin32.sourceforge.net/packages/gzip.htm
-[the platinum searcher]: https://github.com/monochromegane/the_platinum_searcher
-[so-server-unsafe]: http://stackoverflow.com/questions/885793/emacs-error-when-calling-server-start
-[Spacemacs logo]: https://github.com/nashamri/spacemacs-logo
-[Nasser Alshammari]: https://github.com/nashamri
-[compilation-warnings]: https://github.com/quelpa/quelpa/issues/90#issuecomment-137982713
-[cpaulik-unity-icon]: http://splendidabacus.com/posts/2015/03/spacemacs-unity-icon/
-[icon-mac-instructions]: http://www.idownloadblog.com/2014/07/16/how-to-change-app-icon-mac/
-[latest tags]: https://github.com/syl20bnr/spacemacs/tags
-[icon-repository]: https://github.com/nashamri/spacemacs-logo
-[Stack Exchange]: http://emacs.stackexchange.com/questions/tagged/spacemacs
-[Reddit]: https://www.reddit.com/r/spacemacs
-[quote01]: https://gitter.im/syl20bnr/spacemacs?at=568e627a0cdaaa62045a7df6
-[quote02]: https://gitter.im/syl20bnr/spacemacs?at=5768456c6577f032450cfedb
-[build_source]: https://www.gnu.org/software/emacs/manual/html_node/efaq/Installing-Emacs.html
-[Bountysource]: https://salt.bountysource.com/teams/spacemacs
```

Comment:
Your changes to the readme does not look intentional, please remove them.

Reply:
I have come back to the original version and then updated the part concerning company-auctex backend and its relation to company-math.